src/ontology/edits/oeo-social.omn

```diff
@@ -533,6 +535,19 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/370",
         OEO_00030016
     
     
+Class: OEO_00030022
+
+    Annotations: 
+        definition "An organization is an immaterial entity that has the organization role and there exists some sort of legal framework that recognises the entity and its continued existence.",
```

Comment:
Don't we need the reference to 'society' here or does organization become automatically a society via the def of 'organization role'?

Reply:
but we should consider that again when implementing a society class